__tests__/server/utils/logging/production-formatter.spec.js

```diff
@@ -169,6 +169,34 @@ describe('production-formatter', () => {
       expect(() => JSON.parse(entry)).not.toThrowError();
       expect(JSON.parse(entry)).toMatchSnapshot();
     });
+
+    it('encodes ServerSideReported Error as parseable JSON', () => {
+      function buildServerSideErrorEntry() {
+        const error = new Error('something broke');
+        Object.assign(error, {
+          name: 'ServerSideReportedError',
```

Comment:
nitpick ~ might be better to rename this to `type`?

Reply:
@nellyk don't think we can change it, this is part of our internal schema so it can't be changed, I can share details internally 👍

Reply:
FWIW based off of [`Error.prototype.name`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error#instance_properties)